metrics.go

```diff
@@ -122,7 +117,7 @@ func collectMetrics() (*Metrics, error) {
 		metrics.Devices = append(metrics.Devices,
 			&Device{
 				Index:                 strconv.Itoa(index),
-				MinorNumber:           strconv.Itoa(int(minorNumber)),
+				MinorNumber:           strconv.Itoa(index),
```

Comment:
seems you should make index as metric key instead of replace minor value by index

Reply:
> 

Now most of metrics use minor as key, not index id. This PR will fix the Inconsistence for minor and index in some cases (such as multi-A30/A100). Of course it's ok to add new keys like index_id for all metrics who use minor as key but it brings caller who use this exporter a big work and brings low compatibility.